UsefulProteomicsDatabases/ProteinDbLoader.cs

```diff
@@ -515,6 +515,64 @@ public static List<Protein> LoadProteinFasta(string proteinDbLocation, bool onTh
             return result;
         }
 
+        /// <summary>
+        /// Merges proteins of the same accession and sequence. Considers contaminants separately.
+        /// </summary>
+        /// <param name="merge_these"></param>
+        /// <returns></returns>
+        public static IEnumerable<Protein> merge_proteins(IEnumerable<Protein> merge_these)
+        {
+            Dictionary<Tuple<string, string, bool, bool>, List<Protein>> proteins_by_accession_seq_cont_isdecoy = new Dictionary<Tuple<string, string, bool, bool>, List<Protein>>();
```

Comment:
I agree the decoy designation is a bit confusing... That can be inferred from the code.